!SESSION 2023-02-11 09:39:18.660 -----------------------------------------------
eclipse.buildId=4.17.2.202301310704
java.version=17.0.6
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.springframework.boot.ide.branding.sts4
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.springframework.boot.ide.branding.sts4

!ENTRY ch.qos.logback.classic 1 0 2023-02-11 09:39:23.047
!MESSAGE Activated before the state location was initialized. Retry after the state location is initialized.

!ENTRY org.springframework.ide.eclipse.boot.dash.cf 4 0 2023-02-11 09:39:28.101
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.springframework.ide.eclipse.boot.dash.cf [549]
  Unresolved requirement: Require-Bundle: org.springframework.tooling.ls.eclipse.commons
    -> Bundle-SymbolicName: org.springframework.tooling.ls.eclipse.commons; bundle-version="4.17.2.202301310652"; singleton:="true"
       org.springframework.tooling.ls.eclipse.commons [568]
         Unresolved requirement: Require-Bundle: org.springframework.tooling.jdt.ls.commons
           -> Bundle-SymbolicName: org.springframework.tooling.jdt.ls.commons; bundle-version="4.17.2.202301310652"
              org.springframework.tooling.jdt.ls.commons [567]
                No resolution report for the bundle.  Unresolved requirement: Require-Bundle: com.hierynomus.sshj; bundle-version="0.24.0"
    -> Bundle-SymbolicName: com.hierynomus.sshj; bundle-version="0.24.0"
       com.hierynomus.sshj [26]
         Unresolved requirement: Import-Package: org.bouncycastle.crypto.prng; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.crypto.prng; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"; uses:="org.bouncycastle.crypto,org.bouncycastle.crypto.params"
              org.bouncycastle.bcprov [101]
                No resolution report for the bundle.         Unresolved requirement: Import-Package: org.bouncycastle.asn1.nist; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.asn1.nist; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"; uses:="org.bouncycastle.asn1,org.bouncycastle.asn1.x9"
         Unresolved requirement: Import-Package: org.bouncycastle.crypto.ec; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.crypto.ec; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"; uses:="org.bouncycastle.asn1,org.bouncycastle.asn1.x9,org.bouncycastle.crypto,org.bouncycastle.math.ec"
         Unresolved requirement: Import-Package: org.bouncycastle.math.ec; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.math.ec; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"; uses:="org.bouncycastle.math.ec.endo,org.bouncycastle.math.field"
         Unresolved requirement: Import-Package: org.bouncycastle.asn1.x9; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.asn1.x9; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"; uses:="org.bouncycastle.asn1,org.bouncycastle.math.ec"
         Unresolved requirement: Import-Package: org.bouncycastle.util.encoders; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.util.encoders; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"
         Unresolved requirement: Import-Package: org.bouncycastle.jce.provider; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.jce.provider; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"; uses:="javax.crypto,javax.crypto.interfaces,javax.crypto.spec,javax.security.auth.x500,org.bouncycastle.asn1,org.bouncycastle.asn1.pkcs,org.bouncycastle.asn1.x500,org.bouncycastle.asn1.x509,org.bouncycastle.crypto,org.bouncycastle.crypto.params,org.bouncycastle.jcajce,org.bouncycastle.jcajce.provider.config,org.bouncycastle.jcajce.provider.util,org.bouncycastle.jce.exception,org.bouncycastle.jce.interfaces,org.bouncycastle.jce.spec,org.bouncycastle.math.ec,org.bouncycastle.util,org.bouncycastle.x509,org.bouncycastle.x509.util"
         Unresolved requirement: Import-Package: org.bouncycastle.asn1; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.asn1; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"; uses:="org.bouncycastle.util"
         Unresolved requirement: Import-Package: org.bouncycastle.jce.spec; version="[1.57.0,2.0.0)"; resolution:="optional"
           -> Export-Package: org.bouncycastle.jce.spec; bundle-symbolic-name="org.bouncycastle.bcprov"; bundle-version="1.72.0.v20221013-1810"; version="1.72.0"; uses:="org.bouncycastle.asn1.cryptopro,org.bouncycastle.jcajce.spec,org.bouncycastle.jce.interfaces,org.bouncycastle.math.ec"
  Unresolved requirement: Require-Bundle: org.springsource.ide.eclipse.commons.cloudfoundry.client.v2
    -> Bundle-SymbolicName: org.springsource.ide.eclipse.commons.cloudfoundry.client.v2; bundle-version="4.17.2.202301310704"
       org.springsource.ide.eclipse.commons.cloudfoundry.client.v2 [571]
         No resolution report for the bundle.  Unresolved requirement: Require-Bundle: org.bouncycastle.bcprov; bundle-version="1.59.0"
    -> Bundle-SymbolicName: org.bouncycastle.bcprov; bundle-version="1.72.0.v20221013-1810"
  Unresolved requirement: Require-Bundle: org.springframework.ide.eclipse.boot.wizard
    -> Bundle-SymbolicName: org.springframework.ide.eclipse.boot.wizard; bundle-version="4.17.2.202301310704"; singleton:="true"
       org.springframework.ide.eclipse.boot.wizard [556]
         Unresolved requirement: Require-Bundle: org.springframework.ide.eclipse.maven.pom
           -> Bundle-SymbolicName: org.springframework.ide.eclipse.maven.pom; bundle-version="4.17.2.202301310704"; singleton:="true"
              org.springframework.ide.eclipse.maven.pom [561]
                No resolution report for the bundle.  Unresolved requirement: Require-Bundle: org.springframework.tooling.cloudfoundry.manifest.ls
    -> Bundle-SymbolicName: org.springframework.tooling.cloudfoundry.manifest.ls; bundle-version="1.44.0.202301310652"; singleton:="true"
       org.springframework.tooling.cloudfoundry.manifest.ls [565]
         Unresolved requirement: Require-Bundle: org.springframework.tooling.ls.eclipse.commons; bundle-version="4.0.0"
           -> Bundle-SymbolicName: org.springframework.tooling.ls.eclipse.commons; bundle-version="4.17.2.202301310652"; singleton:="true"
  Bundle was not resolved because of a uses constraint violation.
  org.apache.felix.resolver.reason.ReasonException: Uses constraint violation. Unable to resolve resource org.springframework.ide.eclipse.boot.dash.cf [osgi.identity; type="osgi.bundle"; version:Version="4.17.2.202301310704"; osgi.identity="org.springframework.ide.eclipse.boot.dash.cf"; singleton:="true"] because it is exposed to package 'org.bouncycastle.asn1' from resources org.bouncycastle.bcprov [osgi.identity; type="osgi.bundle"; version:Version="1.72.0.v20221013-1810"; osgi.identity="org.bouncycastle.bcprov"] and bcprov [osgi.identity; type="osgi.bundle"; version:Version="1.72.0"; osgi.identity="bcprov"] via two dependency chains.

Chain 1:
  org.springframework.ide.eclipse.boot.dash.cf [osgi.identity; type="osgi.bundle"; version:Version="4.17.2.202301310704"; osgi.identity="org.springframework.ide.eclipse.boot.dash.cf"; singleton:="true"]
    require: (&(osgi.wiring.bundle=org.bouncycastle.bcprov)(bundle-version>=1.59.0))
     |
    provide: osgi.wiring.bundle: org.bouncycastle.bcprov
  org.bouncycastle.bcprov [osgi.identity; type="osgi.bundle"; version:Version="1.72.0.v20221013-1810"; osgi.identity="org.bouncycastle.bcprov"]

Chain 2:
  org.springframework.ide.eclipse.boot.dash.cf [osgi.identity; type="osgi.bundle"; version:Version="4.17.2.202301310704"; osgi.identity="org.springframework.ide.eclipse.boot.dash.cf"; singleton:="true"]
    require: (&(osgi.wiring.bundle=org.bouncycastle.bcpkix)(bundle-version>=1.59.0))
     |
    provide: osgi.wiring.bundle; bundle-version:Version="1.72.0.v20221013-1810"; osgi.wiring.bundle="org.bouncycastle.bcpkix"
  org.bouncycastle.bcpkix [osgi.identity; type="osgi.bundle"; version:Version="1.72.0.v20221013-1810"; osgi.identity="org.bouncycastle.bcpkix"]
    import: (&(osgi.wiring.package=org.bouncycastle.asn1)(&(version>=1.72.0)(!(version>=1.73.0))))
     |
    export: osgi.wiring.package: org.bouncycastle.asn1
  bcprov [osgi.identity; type="osgi.bundle"; version:Version="1.72.0"; osgi.identity="bcprov"]
	at org.eclipse.osgi.container.Module.start(Module.java:463)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1852)
	at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor$1$1.execute(EquinoxContainerAdaptor.java:136)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1845)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1786)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1750)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1672)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)

!ENTRY ch.qos.logback.classic 1 0 2023-02-11 09:42:12.259
!MESSAGE Logback config file: C:\Users\ConradMaré\Desktop\Spring_React\.metadata\.plugins\org.eclipse.m2e.logback\logback.2.1.0.20221015-0744.xml

!ENTRY org.eclipse.jface 2 0 2023-02-11 09:42:15.294
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2023-02-11 09:42:15.294
!MESSAGE A conflict occurred for CTRL+SHIFT+T:
Binding(CTRL+SHIFT+T,
	ParameterizedCommand(Command(org.eclipse.jdt.ui.navigate.open.type,Open Type,
		Open a type in a Java editor,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		WorkbenchHandlerServiceHandler("org.eclipse.jdt.ui.navigate.open.type"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
Binding(CTRL+SHIFT+T,
	ParameterizedCommand(Command(org.eclipse.lsp4e.symbolinworkspace,Go to Symbol in Workspace,
		,
		Category(org.eclipse.lsp4e.category,Language Servers,null,true),
		WorkbenchHandlerServiceHandler("org.eclipse.lsp4e.symbolinworkspace"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2023-02-11 09:42:15.294
!MESSAGE A conflict occurred for ALT+SHIFT+R:
Binding(ALT+SHIFT+R,
	ParameterizedCommand(Command(org.eclipse.jdt.ui.edit.text.java.rename.element,Rename - Refactoring ,
		Rename the selected element,
		Category(org.eclipse.jdt.ui.category.refactoring,Refactor - Java,Java Refactoring Actions,true),
		WorkbenchHandlerServiceHandler("org.eclipse.jdt.ui.edit.text.java.rename.element"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
Binding(ALT+SHIFT+R,
	ParameterizedCommand(Command(org.eclipse.ui.edit.rename,Rename,
		Rename the selected item,
		Category(org.eclipse.ui.category.file,File,null,true),
		WorkbenchHandlerServiceHandler("org.eclipse.ui.edit.rename"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)

!ENTRY org.eclipse.egit.ui 2 0 2023-02-11 09:42:22.583
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\ConradMaré'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.jdt.ui 4 10001 2023-02-11 09:48:35.251
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Error in Java Model (code 969): hoaxify does not exist
	at org.eclipse.jdt.internal.core.JavaElement.newJavaModelException(JavaElement.java:584)
	at org.eclipse.jdt.internal.core.Openable.generateInfos(Openable.java:254)
	at org.eclipse.jdt.internal.core.Openable.openAncestors(Openable.java:528)
	at org.eclipse.jdt.internal.core.Openable.generateInfos(Openable.java:248)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:597)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:328)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:314)
	at org.eclipse.jdt.internal.core.PackageFragmentRoot.getKind(PackageFragmentRoot.java:530)
	at org.eclipse.jdt.ui.wizards.NewContainerWizardPage.initContainerPage(NewContainerWizardPage.java:179)
	at org.eclipse.jdt.ui.wizards.NewContainerWizardPage.initContainerPage(NewContainerWizardPage.java:141)
	at org.eclipse.jdt.ui.wizards.NewPackageWizardPage.init(NewPackageWizardPage.java:169)
	at org.eclipse.jdt.internal.ui.wizards.NewPackageCreationWizard.addPages(NewPackageCreationWizard.java:59)
	at org.eclipse.jface.wizard.WizardDialog.createContents(WizardDialog.java:633)
	at org.eclipse.jface.window.Window.create(Window.java:431)
	at org.eclipse.jface.dialogs.Dialog.create(Dialog.java:1094)
	at org.eclipse.jdt.ui.actions.AbstractOpenWizardAction.run(AbstractOpenWizardAction.java:84)
	at org.eclipse.jdt.internal.ui.wizards.OpenPackageWizardToolbarAction.run(OpenPackageWizardToolbarAction.java:42)
	at org.eclipse.ui.internal.PluginAction.runWithEvent(PluginAction.java:239)
	at org.eclipse.ui.internal.WWinPluginAction.runWithEvent(WWinPluginAction.java:218)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:580)
	at org.eclipse.jface.action.ActionContributionItem.lambda$5(ActionContributionItem.java:453)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4256)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1066)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4054)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3642)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:643)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:550)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.equinox.internal.app.MainApplicationLauncher.run(MainApplicationLauncher.java:36)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1467)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2023-02-11 09:48:35.255
!MESSAGE hoaxify does not exist

!ENTRY org.eclipse.ui 2 2 2023-02-11 09:48:56.092
!MESSAGE Invalid property category path: org.springframework.ide.eclipse.beans.ui.properties.ProjectPropertyPage (bundle: org.springframework.ide.eclipse.xml.namespaces, propertyPage: org.springframework.ide.eclipse.beans.ui.namespaces.projectPropertyPage)

!ENTRY org.springframework.tooling.ls.eclipse.commons 1 0 2023-02-11 09:51:16.677
!MESSAGE executing callback sts4.classpath.QyxGCsfK FAILED

!ENTRY org.springframework.tooling.ls.eclipse.commons 4 0 2023-02-11 09:51:16.681
!MESSAGE TimeoutException
!STACK 0
java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
	at org.springframework.tooling.ls.eclipse.commons.LSP4ECommandExecutor.executeClientCommand(LSP4ECommandExecutor.java:37)
	at org.springframework.tooling.jdt.ls.commons.classpath.SendClasspathNotificationsJob.flush(SendClasspathNotificationsJob.java:178)
	at org.springframework.tooling.jdt.ls.commons.classpath.SendClasspathNotificationsJob.run(SendClasspathNotificationsJob.java:149)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.jface 2 0 2023-02-11 09:55:47.847
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2023-02-11 09:55:47.847
!MESSAGE A conflict occurred for CTRL+SHIFT+C:
Binding(CTRL+SHIFT+C,
	ParameterizedCommand(Command(org.eclipse.tm4e.languageconfiguration.toggleLineCommentCommand,Toggle Line Comment,
		,
		Category(org.eclipse.tm4e.languageconfiguration.category,TM4E Language Configuration,null,true),
		WorkbenchHandlerServiceHandler("org.eclipse.tm4e.languageconfiguration.toggleLineCommentCommand"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.textEditorScope,,,system)
Binding(CTRL+SHIFT+C,
	ParameterizedCommand(Command(org.springframework.tooling.boot.ls.ToggleComment,Toggle Comment,
		,
		Category(org.springframework.ide.eclipse.commands,Spring Generic Text Editor,Spring Language Server Commands,true),
		WorkbenchHandlerServiceHandler("org.springframework.tooling.boot.ls.ToggleComment"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.textEditorScope,,,system)

!ENTRY org.eclipse.lsp4e 4 0 2023-02-11 09:56:46.929
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.lsp4e 4 0 2023-02-11 09:56:46.938
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.lsp4e 4 0 2023-02-11 09:56:46.951
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.lsp4e 2 0 2023-02-11 09:56:56.819
!MESSAGE Could not get semantic tokens due to timeout after 10 seconds
!STACK 0
java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
	at org.eclipse.lsp4e.operations.semanticTokens.SemanticHighlightReconcilerStrategy.fullReconcile(SemanticHighlightReconcilerStrategy.java:280)
	at org.eclipse.lsp4e.operations.semanticTokens.SemanticHighlightReconcilerStrategy.reconcile(SemanticHighlightReconcilerStrategy.java:304)
	at org.eclipse.jface.text.reconciler.MonoReconciler.process(MonoReconciler.java:76)
	at org.eclipse.jface.text.reconciler.AbstractReconciler$BackgroundThread.run(AbstractReconciler.java:207)

!ENTRY org.eclipse.lsp4e 1 0 2023-02-11 09:57:07.685
!MESSAGE [Info]	ESLint server running in node v18.12.1

!ENTRY org.eclipse.jface.text 4 0 2023-02-11 09:57:07.698
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.jface.text 4 0 2023-02-11 09:57:07.699
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.jface.text 4 0 2023-02-11 09:57:07.700
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.jface.text 4 0 2023-02-11 09:57:07.702
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.jface.text 4 0 2023-02-11 09:57:07.703
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.jface.text 4 0 2023-02-11 09:57:07.705
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.lsp4e 4 0 2023-02-11 09:57:07.716
!MESSAGE java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
!STACK 0
java.util.concurrent.CompletionException: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:300)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more
Caused by: java.io.IOException: Cannot run program "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE\.node\node-v18.12.1-win-x64\node.exe" (in directory "C:\Users\ConradMaré\Downloads\sts-4.17.2.RELEASE"): CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.eclipse.lsp4e.server.ProcessStreamConnectionProvider.start(ProcessStreamConnectionProvider.java:54)
	at org.eclipse.wildwebdeveloper.ui.preferences.ProcessStreamConnectionProviderWithPreference.start(ProcessStreamConnectionProviderWithPreference.java:120)
	at org.eclipse.lsp4e.LanguageServerWrapper.lambda$0(LanguageServerWrapper.java:298)
	... 7 more
Caused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more

!ENTRY org.eclipse.lsp4e 1 0 2023-02-11 09:57:07.997
!MESSAGE [Info]	No ES Libary found for file: file:///C:/Users/ConradMar%C3%A9/Desktop/Spring_React/hoaxify/src/main/resources/application.yml

!ENTRY org.eclipse.jdt.ui 4 10001 2023-02-11 10:03:28.948
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Error in Java Model (code 969): com.hoaxify.hoaxify [in src/main/java/com/hoaxify/hoaxify [in hoaxify]] does not exist
	at org.eclipse.jdt.internal.core.MultiOperation.processElements(MultiOperation.java:180)
	at org.eclipse.jdt.internal.core.CopyResourceElementsOperation.processElements(CopyResourceElementsOperation.java:436)
	at org.eclipse.jdt.internal.core.MultiOperation.executeOperation(MultiOperation.java:95)
	at org.eclipse.jdt.internal.core.JavaModelOperation.run(JavaModelOperation.java:740)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2380)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2405)
	at org.eclipse.jdt.internal.core.JavaModelOperation.runOperation(JavaModelOperation.java:811)
	at org.eclipse.jdt.internal.core.JavaModel.runOperation(JavaModel.java:308)
	at org.eclipse.jdt.internal.core.JavaModel.move(JavaModel.java:267)
	at org.eclipse.jdt.internal.core.CompilationUnit.move(CompilationUnit.java:1175)
	at org.eclipse.jdt.internal.corext.refactoring.changes.MoveCompilationUnitChange.doPerformReorg(MoveCompilationUnitChange.java:90)
	at org.eclipse.jdt.internal.corext.refactoring.changes.CompilationUnitReorgChange.perform(CompilationUnitReorgChange.java:68)
	at org.eclipse.ltk.core.refactoring.CompositeChange.perform(CompositeChange.java:281)
	at org.eclipse.jdt.ui.text.java.correction.ChangeCorrectionProposal.performChange(ChangeCorrectionProposal.java:181)
	at org.eclipse.jdt.ui.text.java.correction.ChangeCorrectionProposal.apply(ChangeCorrectionProposal.java:111)
	at org.eclipse.jdt.internal.ui.text.java.hover.AbstractAnnotationHover$AnnotationInformationControl.apply(AbstractAnnotationHover.java:561)
	at org.eclipse.jdt.internal.ui.text.java.hover.AbstractAnnotationHover$AnnotationInformationControl$4.widgetSelected(AbstractAnnotationHover.java:532)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:252)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4256)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1066)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1090)
	at org.eclipse.swt.widgets.Widget.sendSelectionEvent(Widget.java:1107)
	at org.eclipse.swt.widgets.Link.wmNotifyChild(Link.java:725)
	at org.eclipse.swt.widgets.Control.wmNotify(Control.java:5797)
	at org.eclipse.swt.widgets.Composite.wmNotify(Composite.java:1961)
	at org.eclipse.swt.widgets.Control.WM_NOTIFY(Control.java:5365)
	at org.eclipse.swt.widgets.Control.windowProc(Control.java:4799)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:5022)
	at org.eclipse.swt.internal.win32.OS.CallWindowProc(Native Method)
	at org.eclipse.swt.widgets.Link.callWindowProc(Link.java:162)
	at org.eclipse.swt.widgets.Widget.wmLButtonUp(Widget.java:2025)
	at org.eclipse.swt.widgets.Control.WM_LBUTTONUP(Control.java:5178)
	at org.eclipse.swt.widgets.Control.windowProc(Control.java:4780)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:5022)
	at org.eclipse.swt.internal.win32.OS.DispatchMessage(Native Method)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3640)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:643)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:550)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.equinox.internal.app.MainApplicationLauncher.run(MainApplicationLauncher.java:36)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1467)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2023-02-11 10:03:28.949
!MESSAGE com.hoaxify.hoaxify [in src/main/java/com/hoaxify/hoaxify [in hoaxify]] does not exist
